include/eve/concept/transparent.hpp

```diff
@@ -0,0 +1,18 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/traits/transparent.hpp>
+
+#include <concepts>
+
+namespace eve
+{
+  template <typename T>
+  concept transparent_value = !std::same_as<transparent_inner_t<T>, T>;
```

Comment:
This seems like it will return `true` for `unique_ptr`.

Let's do a type trait, with explicit opt ins.

Reply:
How so, there is no ties between our traits ans unique ptr